react_week2_fixed/10.md

```diff
@@ -0,0 +1,17 @@
+## Continue
```

Comment:
Hey @vvennela please make sure this is NOT a hint card. This is activities. This should be all 1.md, 2.md, 3.md, 4.md, no two digit numbers :)